taskt/taskt.csproj

```diff
@@ -217,7 +217,6 @@
     <Compile Include="Core\Automation\Commands\Error Handling Commands\ThrowCommand.cs" />
     <Compile Include="Core\Automation\Commands\Error Handling Commands\TryCommand.cs" />
     <Compile Include="Core\Automation\Commands\Excel Commands\ExcelActivateSheetCommand.cs" />
-    <Compile Include="Core\Automation\Commands\Excel Commands\ExcelAddWorkbookCommand.cs" />
```

Comment:
Please update the references in UIImage.cs for the removed commands.

Reply:
done